plugins/acpi-dmar/fu-plugin-acpi-dmar.c

```diff
@@ -46,6 +46,7 @@ fu_plugin_acpi_dmar_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
 		return;
 	}
 	if (!fu_acpi_dmar_get_opt_in(dmar)) {
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_OEM_FW_UPDATE);
```

Comment:
Not always; I've seen on Lenovo that this is fixable by BIOS config option.

Reply:
Ack, fixed.